site/docs/downloads.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Downloads
+---
+
```

Comment:
Suggest adding some clarification of the various download options and uses:
- source bundle from ASF release distribution site
- binary bundle from ASF release site (includes javadoc, can also access online)
- ASF git repo
- git repo mirror on github

Perhaps something like:

Official Apache Edgent releases are available for download from the ASF distribution site.  A release consists of a source code bundle and a convenience binary bundle.  The Edgent ASF distribution site is --the-dyn-mirror-script-to-incubator/edgent--.  Information about verifying the integrity of the release bundles can also be found there.

The Edgent source is also available from the Edgent ASF git repository --the-url-- or its mirror on github --the-url--.

If you just want to use Edgent, it is easiest to download and unpack a binary bundle.  The bundle includes the release's javadoc.  The javadoc is also accessible online.  maybe simple unpack example here and/or link to getting-started?

A source bundle contains a README describing how to build the sources.

If you want access the latest unreleased Edgent source or to contribute to Edgent runtime development, using the github repository is recommended.  You can also select a particular release version by release tag (e.g., 1.0.0-incubating).  See README.md and DEVELOPMENT.md in the repository for more information.

See --our-contributing-page-url-- for more information about contributing to Edgent development.

----
Of course the above and the information in getting-started, needs to flow well.  No guarantee the above achieves that :-/
----
In the table below I'm wondering if it is better to NOT have links to tgz files.  Might it be better to just supply a link to the dyn-url-incubating/edgent/1.0.0-incubating dir?  Then the README, RELEASE_NOTES, LICENSING files will also be in their face.  Hopefully the "binaries" dir (containing its LICENSE) will be obvious enough.   So combining the Source and Binaries columns into just Distribution Site or such.  Of course all of those files are in the bundles themselves so perhaps I'm just worrying too much about that?  
Adding a release-specific url to the github site seems like it could be helpful? e.g., https://github.com/apache/incubator-edgent/tree/1.0.0-incubating